### PR TITLE
IBM Cloud getInstance support

### PR DESCRIPTION
**Pull request by mhmxs**
New provider is introduced with getInstance support.

### CODE REVIEW

**Comment by mhmxs** [comment]
@keyki please review

**Comment by keyki** [comment]
Can you please provide a test environment? :)

**Comment by mhmxs** [comment]
Could you please explain what do you mean?

File: README.MD [hunk 1]
```diff
@@ -12,7 +12,7 @@ Cloud Haunter (ch) is supports the three major cloud providers.
 Current version of the software has limited features, it can monitor:
  * accesses and keys [AWS, GCP]
  * databases [AWS]
- * instances [AZURE, AWS, GCP]
+ * instances [AZURE, AWS, GCP, IBM]
```
Comment (bergerdenes) [inline README.MD:15]: line 8: four instead of three

**Pull request closed without merging**